sgl-kernel/setup.py

```diff
@@ -49,6 +49,15 @@ def rename_wheel():
     shutil.rmtree(tmp_dir)
 
 
+def update_wheel_platform_tag():
+    wheel_dir = Path("dist")
+    old_wheel = next(wheel_dir.glob("*.whl"))
+    new_wheel = wheel_dir / old_wheel.name.replace(
+        "linux_x86_64", "manylinux2014_x86_64"
```

Comment:
@ByronHsu I believe we should do the same thing for the router wheel as well. Today I noticed that pip doesn't recognize the wheel arch (linux_x86_64) for mariner and tries to build from source. So I had to install it manually.